src/components/DocumentationTopic/OnThisPageSection.vue

```diff
@@ -33,11 +33,17 @@ export default {
       type: String,
       required: true,
     },
+    level: {
+      type: Number,
+      default: 2,
+    },
   },
   created() {
```

Comment:
Right, I think using the `LinkableHeading` could be a fine candidate to register the sections in the beginning. 

In the future if we have live updating of the content, this may cause issues as things will probably get out of order with, duplication and so on.

Reply:
Since #408 has now been merged, this PR will need to be rebased and modified to take those changes into account.

Reply:
I think that makes sense, otherwise we kind of have some components like this which are just leftover from the old implementation and it's more confusing to follow how things work since the markup and registration is now separate. We could potentially do that later though if removing it now is too risky of a change. What do you think?

Reply:
Done, 63aa095c600efef747b029c6a5759214f1fe22df